core/templates/dev/head/pages/creator_dashboard/CreatorDashboard.js

```diff
@@ -124,7 +125,8 @@ oppia.controller('CreatorDashboard', [
 
     $scope.updatesGivenScreenWidth = function() {
       if ($scope.checkMobileView()) {
-        $scope.myExplorationsView = 'card';
+        $scope.myExplorationsView = 
+        constants.ALLOWED_CREATOR_DASHBOARD_DISPLAY_PREFS.get('CARD');
```

Comment:
Does this work? I don't think Object.get() is a function in JS. Should just do constants.ALLOWED_CREATOR_DASHBOARD_DISPLAY_PREFS.CARD, right?

Also, indent this line by 2 from the previous one  -- and consider using parens, e.g.:


    $scope.myExplorationsView = (
      constants.ALLOWED_CREATOR_DASHBOARD_DISPLAY_PREFS.CARD);

Reply:
Done